13_k8s_dev_tools_openlens_k9s_22-sep-2022/README.md

```diff
@@ -63,18 +63,12 @@ Copy password to the clipboard.
 
 10. Run Alliedium [scripts](https://github.com/Alliedium/awesome-linux-config/tree/master/manjaro#instructions).
 
-11. In the second Git Bash instance connected to VM (Steps 5, 6), open file in editor:
+11. In the second Git Bash instance connected to VM (Steps 5, 6) comment the line:
 
 ```
-nano ~/.p10k.zsh
+sed -i "s/typeset -g POWERLEVEL9K_KUBECONTEXT_SHOW_ON_COMMAND='kubectl|helm|kubens|kubectx|oc'/#typeset -g POWERLEVEL9K_KUBECONTEXT_SHOW_ON_COMMAND='kubectl|helm|kubens|kubectx|oc'"
 ```
 
-Search for `kubectl` pattern via \[Ctrl+W\] > enter 'kubectl' > \[Enter\].
-
-Find the line `typeset -g POWERLEVEL9K_KUBECONTEXT_SHOW_ON_COMMAND='kubectl|helm|kubens|kubectx|oc'`.
-
-Comment the line, save changes (\[Ctrl+S\]) and close the editor (\[Ctrl+X\].
-
```

Comment:
may it is better to leave some comment describing what sed command above does, something like 
```
The command above comments out the line typeset -g POWERLEVEL9K_KUBECONTEXT_SHOW_ON_COMMAND='kubectl|helm|kubens|kubectx|oc
```

Reply:
done now, for some reason does not display as [outdated], check here please: https://github.com/Alliedium/devops-course-2022/blob/ALM-2044-final/13_k8s_dev_tools_openlens_k9s_22-sep-2022/README.md